src/components/platform/logout/Logout.tsx

```diff
@@ -3,23 +3,24 @@ import {Session} from "../../../general-components/Session/Session";
 import {RouteComponentProps, withRouter} from "react-router";
 import {reload_app} from "../../../index";
 import {Loader} from "../../../general-components/Loader/Loader";
-import {Messages} from "../../../general-components/Messages/Messages";
+import {Messages, useMessageContext} from "../../../general-components/Messages/Messages";
 
 import "./logout.scss";
 
 
 export function LogoutComponent(props: RouteComponentProps) {
 
+    const {add: addMessage} = useMessageContext();
     const logoutCallback = useCallback(async () => {
         let call = await Session.logout();
 
         if ((call && call.success) || call?.status === 401) {
             reload_app();
 
-            Messages.add("Bis bald!", "SUCCESS", Messages.TIMER);
+            addMessage("Bis bald!", "SUCCESS", Messages.TIMER);
```

Comment:
Message is shown twice while user logs out. also this seems to be a bigger problem (maybe inside loader?)

Reply:
The issues appears, because the developer build of React is used. Since React 18, the Strict mode does mount und unmount every component once before mounting it for real. More on that [here](https://react.dev/reference/react/StrictMode#fixing-bugs-found-by-re-running-effects-in-development). 

I don't think it is useful to fix that here. To be precise, in my opinion, the only useful fix would be to remove the feature of the Loader to call the payload functions. I would change the payload property to accept a promise object, which it subscribes to.

Reply:
Okay, I didn't read the code of the logoutCallback carefully enough. Since the function does perform a network request, I will try to add the above feature without removing the normal payload feature.

Reply:
I added the described feature, but realized that sending the Logout request in componentDidMount or with a function component useEffect, isn't so useful, because it has nothing to do with the visual representation. As a result, I migrated the Logout page to only serve as loader and redirect page.